client/core/helpers.go

```diff
@@ -256,8 +256,9 @@ func (ord *OrderReader) sumTo(filter func(match *Match) bool) uint64 {
 	return v
 }
 
-// hasLiveMatches will be true if there are any matches < MakerRedeemed.
-func (ord *OrderReader) hasLiveMatches() bool {
+// hasActiveMatches will be true if order has any matches that still require some
+// actions to complete settlement.
+func (ord *OrderReader) hasActiveMatches() bool {
```

Comment:
Terms **live** and **active** seem to be used interchangeably, it isn't obvious though so I'd prefer to have single name for the same thing when possible, hence renaming to **active**.